### ISSUE TITLE
Running Gradle task 'assembleDebug'... Takes to Much time and do nothing

### ISSUE THREAD

**Issue by abdullahshamroz** [flutter/flutter#54197]
I am trying to run a basic flutter project (which comes after creating new flutter project "which is basically a counter) while connecting my LG G6 (android 8.0) because running an emulator hangs my laptop (Dell Inspiron 15). I am running on ubuntu 18.04 LTS. 

**Problem (screenshot attached):**
I got the application logo in my mobile too, while debugging the code, I get following two lines in 
my debug console and the "Running Gradle task is taking too much time (while my mobile screen is white at running this thing) and I've given it straight 5 hours and it has done nothing but only loading.."   
 
![flutter running gradle problem](https://user-images.githubusercontent.com/43386836/78682919-0e9b9d00-7908-11ea-89b4-7d58392ca0f5.png)

This is the result of my **" flutter doctor -v "** :

```
[✓] Flutter (Channel stable, v1.12.13+hotfix.9, on Linux, locale en_US.UTF-8)
    • Flutter version 1.12.13+hotfix.9 at /home/jack/Documents/development/flutter
    • Framework revision f139b11009 (8 days ago), 2020-03-30 13:57:30 -0700
    • Engine revision af51afceb8
    • Dart version 2.7.2

 
[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.3)
    • Android SDK at /home/jack/Android/Sdk
    • Android NDK location not configured (optional; useful for native profiling support)
    • Platform android-29, build-tools 29.0.3
    • Java binary at: /home/jack/Documents/development/android studio/android-studio/jre/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_212-release-1586-b4-5784211)
    • All Android licenses accepted.

[✓] Android Studio (version 3.6)
    • Android Studio at /home/jack/Documents/development/android studio/android-studio
    • Flutter plugin version 44.0.2
    • Dart plugin version 192.7761
    • Java version OpenJDK Runtime Environment (build 1.8.0_212-release-1586-b4-5784211)

[✓] VS Code (version 1.43.2)
    • VS Code at /usr/share/code
    • Flutter extension version 3.9.1

[✓] Connected device (1 available)
    • LGM G600L • LGMG600Ld3f94e75 • android-arm64 • Android 8.0.0 (API 26)

• No issues found!
```

**Comment by VladyslavBondarenko** [comment]
Hi @abdullahshamroz 
what Linux kernel version do you have?
is it your case https://github.com/flutter/flutter/issues/49185?
could you try if are you experiencing this with dev or master channel?

**Comment by NicolasD8** [comment]
Hi @abdullahshamroz 
i am experiencing the same problem but on windows can anyone help please!!

**Comment by VladyslavBondarenko** [comment]
@NicolasD8 please consider to create a separate issue for this

**Comment by abdullahshamroz** [comment]
> Hi @abdullahshamroz
> what Linux kernel version do you have?
> is it your case #49185?
> could you try if are you experiencing this with dev or master channel?

Hi, thanks for the help, I got it resolved after updating some sdk plugins and avd manager in android studio and after then I ran " flutter run " command in my vs terminal and the flutter demo homepage ran on my connected device.

**Comment by abdullahshamroz** [comment]
> Hi @abdullahshamroz
> i am experiencing the same problem but on windows can anyone help please!!

Hi, thanks for the help, I got it resolved after updating some sdk plugins and avd manager in android studio and after then I ran " flutter run " command in my vs terminal and the flutter demo homepage ran on my connected device. 

check out if there are any plugin updates available (dart or flutter). 
Open Android Studio  > Tools > AVD manager and check for updates. 
Then Open Android Studio  > Tools > SDK manager and do the same. 
Save and run your program and I hope it might work.
